spring-cloud-dataflow-server-kubernetes-docs/src/main/asciidoc/getting-started.adoc

```diff
@@ -384,3 +384,81 @@ dataflow:>task execution list
+
+=== Memory and CPU Settings
+
+The Spring Cloud Deployer that is being used has certain defaults for the memory and CPU setting that are used for deployed apps. With the defaults the container Limits and Requests are set to the following:
+
+```
+    Limits:
+      cpu:	500m
+      memory:	512Mi
+    Requests:
+      cpu:	500m
+      memory:	512Mi
+```
+
+You might find that the 512Mi memory limit is too low and to increase it you can provide a common `spring.cloud.deployer.memory` deployer property like this (replace <app> with the name of the app you would like to set this for):
+
+```
+deployer.<app>.memory=640m
+```
+
+This property affects bot the Requests and Limits memory value set for the container.
+
+If you would like to set the Requests and Limits values separately you would have to use the deployer properties that are specific to the Kubernetes deployer. To set the Limits to 1000m for cpu, 1024Mi for memory and Requests to 800m for cpu, 640Mi for memory you can use the following properties:
+
+```
+deployer.<app>.kubernetes.limits.cpu=1000m
+deployer.<app>.kubernetes.limits.memory=1024Mi
+deployer.<app>.kubernetes.requests.cpu=800m
+deployer.<app>.kubernetes.requests.memory=640Mi
+```
+
+That should result in the following container settings being used:
+
+```
+    Limits:
+      cpu:	1
+      memory:	1Gi
+    Requests:
+      cpu:	800m
+      memory:	640Mi
+```
+
+NOTE: When using the common memory property you should use and `m` suffix for the value while when using the Kubernetes specific properties you should use the Kubernetes `Mi` style suffix.
+
+The settings we have used so far only affect the settings for the container, they do not affect the memory setting for the JVM process in the container. If you would like to set JVM memory settings you can provide an environment variable for this, see the next section for details.
+
+=== Environment Variables
+
+To set an environment variable for a deployed app use the `spring.cloud.deployer.kubernetes.environmentVariables` property like this:
+
+```
+deployer.<app>.kubernetes.environmentVariables=JAVA_TOOL_OPTIONS=-Xmx1024m
+```
+
+This example sets the `JAVA_TOOL_OPTIONS` environment variable to pass in a JVM memory setting. 
+
+=== Liveness and Readiness Probes
+
+The liveness and readiness probes are using the paths `\health` and `\info` respectively. They use a delay of 10 for both and a period of 60 and 10 respectively. You can chage these defaults when you deploy by using deployer properties.
+
+Here is an example changing the liveness probe (just replace <app> with the name of your app):
+
+```
+deployer.<app>.kubernetes.livenessProbePath=/info
+deployer.<app>.kubernetes.livenessProbeDelay=120
+deployer.<app>.kubernetes.livenessProbePeriod=20
+```
+
+For setting values for the readiness probe just change the property name to include readiness instead of liveness.
```

Comment:
Maybe:

"Similarly, swap "liveness" for "readiness" to override the default readiness settings."